src/contracts/pods/EigenPod.sol

```diff
@@ -280,6 +289,41 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         }
     }
 
+    function submitPartialWithdrawalsBatchForVerification(
+        uint64 oracleTimestamp,
+        uint64 startTimestamp,
+        uint64 endTimestamp,
+        bytes32 FUNCTION_ID
+    ) external {
+        eigenPodManager.requestProofViaFunctionGateway(
+            FUNCTION_ID, 
+            startTimestamp,
+            endTimestamp,
+            address(this),
+            oracleTimestamp,
+            requestNonce,
+            this.handleCallback.selector
+        );
+        /**
+        * Zero out the running total of partial withdrawals claimed.  Once another merkle proof for partial withdrawals is proven
+        * this running total wil once again start accumulating.
+        */
+        sumOfPartialWithdrawalsClaimedGwei = 0;
```

Comment:
This doesn't seem right at all. Shouldn't we be checking the reported amount in the callback function and decreasing this (+ the amount of ETH forwarded on) as appropriate?